### DISCUSSION TITLE
无法使用退款

### DISCUSSION THREAD

**Discussion by waptomcat** [getlantern/lantern-forum-cn#754] [General]
昨天刚充值的PRO，无论安卓还是pc均无法用邮箱登陆，接到验证码但是安卓和pc输入完验证码就闪退

**Comment by waptomcat** [comment]
![mmexport1722828759364](https://github.com/user-attachments/assets/75be3a9a-c11f-4851-aae8-96aec913f6c2)

**Reply by waptomcat** [reply]
怎么没声音了，给我个回信

**Comment by MaxMKMax** [comment]
@waptomcat 您好，谢谢您的反馈。我会帮您申请退款. 但需要您帮一个忙! 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Reply by waptomcat** [reply]
![Uploading Screenshot_20240806-124009.png…]()

**Reply by waptomcat** [reply]
今天还是只要输入验证码就闪退，已反馈，请退款，账号邮箱wapcat78@gmail.com

**Reply by waptomcat** [reply]
![Uploading Screenshot_20240806-124444.png…]()

**Reply by waptomcat** [reply]
RE: cannot login {75450585}

**Reply by waptomcat** [reply]
请给我个回信

**Comment by waptomcat** [comment]
为什么不回

**Comment by waptomcat** [comment]
为什么一直不回

**Reply by liuh23** [reply]
> 为什么一直不回

请问你的退款，退给你了吗。我这边一直没给我退

**Comment by MaxMKMax** [comment]
@waptomcat 不好意思. 不知道为什么您的帖子没弹上来. 没看到. 非常抱歉. 好的我帮您提交退款申请. 谢谢!

**Comment by MaxMKMax** [comment]
@waptomcat 您好. 已经帮您弄退款了. 在进行当中. 谢谢您的耐心等待. 过 2 周请您查看银行和邮箱..
